charts/gardener-extension-shoot-lakom-service/templates/service.yaml

```diff
@@ -10,6 +10,10 @@ metadata:
   namespace: {{ .Release.Namespace }}
   annotations:
     networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.controllers.serverPort }}}]'
+    networking.resources.gardener.cloud/from-all-seed-scrape-targets-allowed-ports: '[{"port":{{ .Values.controllers.metricsPort }},"protocol":"TCP"}]'
+    # TODO: This label approach is deprecated and no longer needed in the future. Remove them as soon as gardener/gardener@v1.75 has been released.
```

Comment:
Do you mean annotation, not labels?
And as it is deprecated, why is it added now and needs to be removed in several weeks again?

Reply:
Yes, annotations of course, fixed it.
I think these annotations are already missing right now (when Gardener's `FullNetworkPolicyInRuntimeCluster` feature gate is enabled, `seed-prometheus` cannot talk to this extension without them).